lib/backpex/resource.ex

```diff
@@ -503,6 +503,8 @@ defmodule Backpex.Resource do
     {:ok, item}
   end
 
+  defp broadcast({:ok, item} = event, _event, nil) do: event
+
   defp broadcast({:error, _reason} = event, _event, _pubsub), do: event
```

Comment:
```suggestion
  defp broadcast(result, _event, _pubsub), do: result
```

We can handle both cases with one function clause.